querydsl-sql-spatial/pom.xml

```diff
@@ -109,19 +109,20 @@
       <groupId>org.postgis</groupId>
       <artifactId>postgis-jdbc</artifactId>
       <version>1.3.3</version>
+      <scope>compile</scope>
       <optional>true</optional>
     </dependency>
     <dependency>
       <groupId>com.oracle</groupId>
       <artifactId>ojdbc6</artifactId>
       <version>${oracle.version}</version>
-      <optional>true</optional>
+      <scope>test</scope>
```

Comment:
Database drivers are only required in the test scope. No need to declare them optional in the compile scope. Obviously users are free to use whatever driver they want, but QueryDSL is completly driver agnostic.